### PR TITLE
FEAT: Adding EbaySplitButton component

### PR DESCRIPTION
**Pull request by shpandian**
1. adding ebay-split-button
2. updating call back from main 
3.updating stories

### CODE REVIEW

File: src/ebay-menu/types.ts [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+
 export type EbayMenuType = 'radio' | 'checkbox'
 export type EbayMenuPriority = 'primary' | 'secondary' | 'none'
+
+type ContainerDivProps = Omit<ComponentProps<'div'>, 'onKeyDown' | 'onChange' | 'onSelect'>
+type ContainerSpanProps = Omit<ComponentProps<'span'>, 'onKeyDown' | 'onChange' | 'onSelect'>
+
+export type SelectCallback = (i: number, checked: boolean) => void
+export type ChangeCallback = (i: number, checked: boolean) => void
+export type KeyDownCallback = (i: number, checked: boolean) => void
```
Comment (shpandian) [inline src/ebay-menu/types.ts:11]: @HenriqueLimas i see main branch also has support for checkedindexes array, its better to move the API to that. what do u think?

File: src/ebay-menu-button/__tests__/index.stories.tsx [hunk 2]
```diff
@@ -177,7 +177,11 @@ storiesOf('ebay-menu-button', module)
                 text={`Sort order: ${sortItems[checkedSort]}`}
                 type="radio"
                 checked={defaultSortIndex}
-                onChange={(index) => setCheckedSort(index)}
+                onChange={(index) => { 
+                    console.log('am i here')
```
Comment (HenriqueLimas) [inline src/ebay-menu-button/__tests__/index.stories.tsx:181]: We can remove this console 🙃

Comment (shpandian) [reply src/ebay-menu-button/__tests__/index.stories.tsx:181]: great catch :)

Comment (shpandian) [reply src/ebay-menu-button/__tests__/index.stories.tsx]: fixed

**Review by HenriqueLimas — APPROVED** [review]

**Pull request merged**